app/app.controllers.js

```diff
@@ -16,27 +17,36 @@ exports.apiDescription = (req, res, next) => {
   });
 };
 
-    
-
-exports.getAllTopics = ( req,res,next)=>{
-     selectTopics().then((topics)=>{
-        res.status(200).send({topics});
-    }).catch(next)
-}
-
+exports.getAllTopics = (req, res, next) => {
+  selectTopics()
+    .then((topics) => {
+      res.status(200).send({ topics });
+    })
+    .catch(next);
+};
 
 exports.getArticle = (req, res, next) => {
-selectArticle()
-      .then((articles) => {
-        res.status(200).send({ articles });
-      })
-      .catch(next);
-  }
-  exports.getArticleById = (req, res, next) => {
-    const { article_id } = req.params;
-    selectArticlesById(article_id)
-      .then((article) => {
-        res.status(200).send(article);
-      })
-      .catch(next);
-  }; 
+  selectArticle()
+    .then((articles) => {
+      res.status(200).send({ articles });
+    })
+    .catch(next);
+};
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  selectArticlesById(article_id)
+    .then((article) => {
+      res.status(200).send(article);
+    })
+    .catch(next);
+};
+
+exports.getCommentsByArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  selectComment(article_id)
+    .then((article) => {
+      res.status(200).send(article);
```

Comment:
We should be sending our response back in an object 👀  - what are we sending back? Would this be more semantic if it was on a key of comments?